### PR TITLE
chore: remove web.link macro - es

### LINKED ISSUE #9076: chore: remove web.link macro - es
Remove the `web.link` macro from ES docs.

### PR DESCRIPTION
**Pull request by schalkneethling**
Remove the web.link macro from ES docs.

fix #9076

### CODE REVIEW

File: files/es/web/html/element/q/index.html [hunk 1]
```diff
@@ -26,24 +26,24 @@
 <table class="properties">
  <tbody>
   <tr>
-   <th scope="row">{{web.link("/es/docs/Web/HTML/Content_categories", "Categorías de contenido")}}</th>
-   <td>{{web.link("/es/docs/Web/HTML/Content_categories#Flow_content", "Flujo de contenido")}}, {{web.link("/es/docs/Web/HTML/Content_categories#Phrasing_content", "redacción de contenido")}}, contenido palpable.</td>
+   <th scope="row"><a href="/es/docs/Web/HTML/Content_categories">Categorías de contenido</a>{{</th>
```
Comment (cw118) [inline files/es/web/html/element/q/index.html:29]: It seems that these `{{` are causing the PR test to fail (to the `es` team, I hope it's OK for me to comment here :))

```suggestion
   <th scope="row"><a href="/es/docs/Web/HTML/Content_categories">Categorías de contenido</a></th>
```

**Comment by cw118** [comment]
@schalkneethling Hello, just jumping in to comment on `conflicting` cleanup :)

When conflicting files are removed, we also need to fix/update the redirects. Here, (I believe) `/es/docs/MDN/Contribute/Structures/Macros` should have a redirect to `/es/docs/MDN/Structures/Macros` which can be done with `yarn content add-redirect /es/docs/MDN/Contribute/Structures/Macros /es/docs/MDN/Structures/Macros` (or `yarn tool`). The check redirects workflow fails because the redirect points to the deleted conflicting file instead.

Let me know if I can help in any way, I have started on `conflicting`/`orphaned` documentation and will create a PR for that as soon as possible :)

**Comment by schalkneethling** [comment]
> `yarn content add-redirect /es/docs/MDN/Contribute/Structures/Macros /es/docs/MDN/Structures/Macros` (or `yarn tool`).

Thanks! I guess I can still run this now, even though I have already deleted the page?

**Comment by cw118** [comment]
> Thanks! I guess I can still run this now, even though I have already deleted the page?

It should still work yes :) (if it does not for whatever reason, we can find a way to add the redirect manually instead)

**Comment by schalkneethling** [comment]
> > Thanks! I guess I can still run this now, even though I have already deleted the page?
> 
> It should still work yes :) (if it does not for whatever reason, we can find a way to add the redirect manually instead)

Wait, are you referring to `https://github.com/mdn/translated-content/pull/9077/files#diff-896db88fc652f2892a64a03601e0e18ed654ced2944d98ac57bafb680240549e`? There is no longer an index page inside macros so, perhaps this needs to be redirected to `docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros`

So, redirect `/es/docs/MDN/Contribute/Structures/Macros` to `/es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros`

**Comment by cw118** [comment]
> So, redirect /es/docs/MDN/Contribute/Structures/Macros to /es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros

Yes this is the correct link/redirect. Mea culpa, I should have checked 😖

**Comment by schalkneethling** [comment]
> > So, redirect /es/docs/MDN/Contribute/Structures/Macros to /es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros
> 
> Yes this is the correct link/redirect. Mea culpa, I should have checked 😖

No worries, thanks for your help.

**Comment by cw118** [comment]
I just remembered another potential issue with adding this redirect: if the page doesn't exist for the `es` locale, the redirect will still be "invalid" according to yari. 

So, either we ask the Spanish team to translate/add the page *or* we move the old conflicting page to the slug `MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros` for now (and l10n-es contributors can update it in the future).

**Comment by schalkneethling** [comment]
> I just remembered another potential issue with adding this redirect: if the page doesn't exist for the `es` locale (i.e. if `/es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros` does not exist), the redirect will still be "invalid" according to yari.
> 
> So, either we ask the Spanish team to translate/add the page _or_ we move the old conflicting page to the slug `MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros` for now (and l10n-es contributors can update it in the future).

Screenshot 2022-10-07 at 23 58 24

That is all good, the file does exist.

**Comment by schalkneethling** [comment]
@cw118 Do you install Yari globally, or how do you run commands inside the translated content repo? There is no `package.json` and I cannot use `npx` for the same reason. I assume if I run this inside the `mdn/content` repo it will add the redirect to the `_redirects.txt` file there. I _do_ see there is a redirects file inside the `es` folder. Thanks!

**Comment by cw118** [comment]
> @cw118 Do you install Yari globally, or how do you run commands inside the translated content repo? There is no `package.json` and I cannot use `npx` for the same reason.

I change to my `content` or `yari` directory to run commands. For `add-redirect`, we use the full slug which includes `/<locale>/docs` and for `delete`/`move` we provide the locale as an argument so that yari makes the changes on translated-content.

> I assume if I run this inside the `mdn/content` repo it will add the redirect to the `_redirects.txt` file there. I _do_ see there is a redirects file inside the `es` folder.

Exactly, and this will work with yari too (if I correctly understood what you mean) :)

**Comment by cw118** [comment]
@schalkneethling Can I help in any way, e.g. add the redirect? The link `/es/docs/MDN/Contribute/Structures/Macros` still points to the conflicting page `/es/docs/conflicting/MDN/Writing_guidelines/Page_structures_978c99f4f82eae196d51ce675e5181c7` when it should instead redirect to `/es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros` (cf. [the failed CI](https://github.com/mdn/translated-content/actions/runs/3210107829/jobs/5247324800#step:6:18))

**Comment by schalkneethling** [comment]
> @schalkneethling Can I help in any way, e.g. add the redirect? The link `/es/docs/MDN/Contribute/Structures/Macros` still points to the conflicting page `/es/docs/conflicting/MDN/Writing_guidelines/Page_structures_978c99f4f82eae196d51ce675e5181c7` when it should instead redirect to `/es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros` (cf. [the failed CI](https://github.com/mdn/translated-content/actions/runs/3210107829/jobs/5247324800#step:6:18))

I am looking at adding the redirect now; thank you for the offer to help, though. I just might take you up on that :)

**Comment by schalkneethling** [comment]
> @schalkneethling Can I help in any way, e.g. add the redirect? The link `/es/docs/MDN/Contribute/Structures/Macros` still points to the conflicting page `/es/docs/conflicting/MDN/Writing_guidelines/Page_structures_978c99f4f82eae196d51ce675e5181c7` when it should instead redirect to `/es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros` (cf. [the failed CI](https://github.com/mdn/translated-content/actions/runs/3210107829/jobs/5247324800#step:6:18))

Added.

File: files/es/_redirects.txt [hunk 2]
```diff
@@ -1222,7 +1222,7 @@
 /es/docs/MDN/Contribute/Procesos	/es/docs/MDN/Contribute/Processes
 /es/docs/MDN/Contribute/Structures	/es/docs/conflicting/MDN/Writing_guidelines/Page_structures
 /es/docs/MDN/Contribute/Structures/Ejemplos_ejecutables	/es/docs/conflicting/MDN/Writing_guidelines/Page_structures_479b8c93d0b37a3f2a020112ae60e692
-/es/docs/MDN/Contribute/Structures/Macros	/es/docs/conflicting/MDN/Writing_guidelines/Page_structures_978c99f4f82eae196d51ce675e5181c7
+/es/docs/MDN/Contribute/Structures/Macros	/es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros
 /es/docs/MDN/Contribute/Structures/Macros/Commonly-used_macros	/es/docs/MDN/Writing_guidelines/Page_structures/Macros/Commonly_used_macros
```
Comment (schalkneethling) [inline files/es/_redirects.txt:1226]:
```suggestion
```

**Comment by cw118** [comment]
Hmm, it looks like there are multiple redirects pointing to the conflicting page that I did not notice 😅 We should probably search  for `/es/docs/conflicting/MDN/Writing_guidelines/Page_structures_978c99f4f82eae196d51ce675e5181c7` in the `es` redirects and update all of those as well

**Comment by schalkneethling** [comment]
> Hmm, it looks like there are multiple redirects pointing to the conflicting page that I did not notice 😅 We should probably search for `/es/docs/conflicting/MDN/Writing_guidelines/Page_structures_978c99f4f82eae196d51ce675e5181c7` in the `es` redirects and update all of those as well

Yeah, I removed one but it looks like there are more. Let me search and replace.

**Comment by cw118** [comment]
Sorry for all of the mistakes :/ There are some tips that I will make sure to add to the conflicting/orphaned documentation...

**Comment by schalkneethling** [comment]
@cw118 Alrighty, we are green.

File: files/es/learn/html/introduction_to_html/getting_started/index.md [hunk 3]
```diff
@@ -677,8 +702,7 @@ En los ejemplos anteriores se han incluido espacios en blanco y saltos de línea
 ```html
 <p>Los perros son tontos.</p>
 
-<p>Los    perros        son
-         tontos.</p>
+<p>Los perros son tontos.</p>
```
Comment (Graywolf9) [inline files/es/learn/html/introduction_to_html/getting_started/index.md:705]: This seems to be intentional for the example

```suggestion
<p>Los    perros        son
         tontos.</p>
```

File: files/es/learn/html/introduction_to_html/getting_started/index.md [hunk 4]
```diff
@@ -704,7 +728,8 @@ El equivalente de referencia de caracter podría recordarse fácilmente porque e
 Considera los dos siguientes párrafos:
 
 ```html
-<p>En HTML, defines un párrafo con el elemento <p>.</p>
+<p>En HTML, defines un párrafo con el elemento</p>
+<p>.</p>
```
Comment (Graywolf9) [inline files/es/learn/html/introduction_to_html/getting_started/index.md:732]: It seems to be intentional for the example

```suggestion
<p>En HTML, defines un párrafo con el elemento <p>.</p>
```

File: files/es/learn/html/introduction_to_html/html_text_fundamentals/index.md [hunk 5]
```diff
@@ -276,10 +292,7 @@ Cada lista desordenada comienza con un elemento {{HTMLElement("ul")}} («_unorde
 
 ```html
 <ul>
-leche
-huevos
-pan
-hummus
+  leche huevos pan hummus
```
Comment (Graywolf9) [inline files/es/learn/html/introduction_to_html/html_text_fundamentals/index.md:295]: this was like that to be a readable example

```suggestion
leche
huevos
pan
hummus
```

**Review by Graywolf9 — COMMENTED** [review]
Thank you so much for your time and contribution @schalkneethling !

Also thank you so much for the guide to the conflicting content @cw118 !

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for your time and contribution!

**Pull request merged**